README.md

```diff
@@ -1,58 +1,195 @@
 # ManageIQ RPM Build
 
-This repository contains code to build RPMs for ManageIQ appliances and container images:
+## Summary
+
+This repository contains code to build RPMs for ManageIQ appliances and container images.
+
+It performs the following tasks:
 
   * Clone source repos
   * Run bundle install, compiles assets for UI and a few other tasks needed
   * Create tarballs
   * Build RPMs (locally in the container image or in Copr)
 
+## Output
+
+It produces the following rpms:
+
+| rpm                        | contents                                                          |
+|----------------------------|-------------------------------------------------------------------|
+| `manageiq-appliance-tools` | **empty**                                                         |
+| `manageiq-gemset`          | `/opt/manageiq/manageiq-gemset/` gems, providers, ui-classic      |
+| `manageiq-gemset-services` | `manageiq-provider-*{.target,@.service}`                          |
+| `manageiq-core`            | `vmdb/{app,certs,content,lib,product,systemd}`                    |
+| `manageiq-core-services`   | `manageiq-*{.target,@.service}`                                   |
+| `manageiq-system`          | `/etc/{default,chron}, /usr/bin/evm*, system/{evm,miq}*.service`  |
+| `manageiq-appliance`       | `/etc/httpd/conf.d/manageiq-*, /etc/motd.manageiq`                |
+| `manageiq-ui`              | `vmdb/public/{assets,packs,ui}`                                   |
+| `manageiq-pods`            | **empty**                                                         |
+
+* `/usr/lib/systemd/system/` contains systemd service files.
+* `/var/www/miq/vmdb` contains vmdb files.
+* `/opt/manageiq/manageiq-appliance` contains files linked into `/etc/`.
+* `/opt/manageiq/manifest` contains manifests.
+
 ## Steps
 
-1. Build container image
+## Obtain container base image
+
+You can either downloading the latest build image or building a new one locally:
+
+  - `docker pull manageiq/rpm_build:latest`
+  - `docker pull manageiq/rpm_build:latest-jansa`
+  - `docker build --pull --tag $USER/rpm_build:latest .`
+
+If you modify files in in this respository like adding a file to the rpm, it is necessary to rebuild this image as shown by the third example.
+Do keep in mind that having a local `Gemfile.lock` before running `docker build` can cause versioning issues with bundler.
```

Comment:
Should we dockerignore Gemfile.lock?

Reply:
Maybe?  In production builds we _want_ to bring a Gemfile.lock during rpm build because we want to ensure we aren't arbitrarily updating gems.  We usually have a Gemfile.lock.release file, but I don't know how @simaishi made that be the "real" lock file.

Reply:
Oh I misunderstood this - I was thinking the Gemfile.lock was the _application's_ Gemfile.lock as opposed to the one for manageiq-rpm_build itself.  Yeah, for the latter, we can probably .dockerignore that.